### PR TITLE
OSDOCS-4927: Release note for node port issue

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.12 (publishing with z-stream, 4.12.1)

Issue:
https://issues.redhat.com/browse/OSDOCS-4926

Link to docs preview:
https://55214--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-12-release-notes.html#microshift-known-issues
(It is the last bullet in this section)

QE review:
Developer Preview software; SME ack required
No QE required

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@pacevedom or @jogeo PTAL and let me know if lgtm or needs edits.

**Review by pacevedom — APPROVED** [review]
/lgtm

File: microshift_release_notes/microshift-4-12-release-notes.adoc [hunk 1]
```diff
@@ -57,6 +57,8 @@ And reload the daemon.
 
 * There is an issue with the way {product-title}'s CNI driver manages the firewall rules. If the firewall settings are modified after {product-title} is started then {product-title} must be restarted. The recommended workaround is to configure firewall rules before starting {product-title}.
 
+* There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To work around this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
```
Comment (snarayan-redhat) [inline microshift_release_notes/microshift-4-12-release-notes.adoc:60]: `installation` instead of `install`

Comment (snarayan-redhat) [inline microshift_release_notes/microshift-4-12-release-notes.adoc:60]:
```suggestion
* In the new installations of {product-title}, the `NodePort` service traffic might be stopped. To work around this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
```
Just a suggestion, based on the my understanding, to keep the sentence minimal. Feel free to ignore if the suggestion doesn't technically sounds correct.

**Review by snarayan-redhat — COMMENTED** [review]
great work! Couple of nits and a suggestion.

**Pull request merged**